databases/qdrant/docker/chatbot/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.11-slim-bookworm
```

Comment:
Could you add to this file a step that simply builds the Dockerfile (just a `docker build`) so we have this added check?

https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/workflows/databases-qdrant-ci.yml

Reply:
Also note that Renovate will immediately want to change this Python image version to 3.12

Reply:
As i remember some used here python libraries required to use python 3.11, can't say which ones right now

Reply:
Required exactly 3.11, or "at least 3.11"?
In either case that's one more reason why I'm suggesting at minimum a "docker build" workflow so that we have at least that check having to pass if Renovate decides to bump any versions